howso/direct/client.py

```diff
@@ -722,6 +723,8 @@ def create_trainee(  # noqa: C901
             (Not implemented in this client)
         resources : dict, optional
             (Not implemented in this client)
```

Comment:
Lets add deprecation to the docstring for resources and library_type

Reply:
Done.  Also copied this doc string to `AbstractHowsoClient` and included the deprecation warnings there too.